pages/linux/fdisk.md

```diff
@@ -2,6 +2,7 @@
 
 > A program for managing partition tables and partitions on a hard disk.
 > See also: `partprobe`.
+> More information: <https://manned.org/fdisk>.
```

Comment:
```suggestion
> More information: <https://www.redhat.com/sysadmin/partitions-fdisk>.
```

Reply:
the fdisk manual really provides little actual information, and is just a really complicated guide, it was tldr

Reply:
Well, at least it shows all of the command line options. Personally I'd prefer to get an overview in the man page and then look if I find any useful tutorials

Reply:
Sometimes I wonder whether we should have a man page link and a more help link

Reply:
I think we discussed that in depth in #5510. Generally and especially in this case I prefer the man page, since I don't like just copy-pasting commads from blogs without knowing what the command can do else. The man page gives you an overview on what the command can do and then you can search for tutorials for those flags if you want to.

Reply:
I wouldn't ever suggest a random page from sketchywebpage.com, its from redhat.com and its one of their guides

Reply:
If there are certain things you have to pay attention to, shouldn't we include it in the page itself insead of linking to a page which shows how to solve it?

Reply:
I have another thought on it, this page is on the Linux category.

Most, if not all. distros have a man page, making this just a duplicate of what everyone using this command has in their offline terminal already

Reply:
> Most, if not all. distros have a man page, making this just a duplicate of what everyone using this command has in their offline terminal already

As far as I understand, the goal of tldr is just to provide an alternate way to consume documentation. The entire point is that it "duplicates" the information, and that's 100% fine.

I think in this case, manned is better for supplementary information:
* it provides more information (often first party)
* man pages will be kept up-to-date, articles like this usually aren't revised after publication
* it's more consistent with other pages (lots of manned links!)
* man pages are standardized (in the same ways tldr pages are standardized) so it's immediately familiar

I'm not against linking to external sources like articles, but I think we should prefer man pages, the link to the repository, or even the README over arbitrary articles regardless of reputability.